rapids_config.cmake

```diff
@@ -11,6 +11,10 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
+
+set(rapids-cmake-repo bdice/rapids-cmake)
+set(rapids-cmake-branch nvcomp-4.1.0.6)
+
```

Comment:
Reverting these changes, since CI passed. We need to keep the remaining changes and merge them.
```suggestion
```